.travis.yml

```diff
@@ -1,18 +1,22 @@
-sudo: required
 language: java
 jdk:
-- oraclejdk8
+  - oraclejdk8
+
 before_cache:
-- rm -fr $HOME/.gradle/caches/
+  - rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
+  - rm -fr $HOME/.gradle/caches/*/plugin-resolution/
```

Comment:
May I ask is there anything else you would like to keep from $HOME/.gradle/caches

Reply:
Just going on Travis recommendations, as it is its throwing away everything, so pretty pointless.